scania-digital-design-system/tegel
# fix(card/banner/toast): align components

Aligns the slots in the card, banner and toast components.
All components now have slots **and** prop for header and subheader.

**Solving issue**  
Fixes: #

**How to test**  
1. Go to storybook
2. Check in Components -> Card/Toast/Banner
3. Check the API and the subheader slot in particular.
